config/datatypes_conf.xml.sample

```diff
@@ -744,6 +746,8 @@
     <sniffer type="galaxy.datatypes.tabular:Sam"/>
     <sniffer type="galaxy.datatypes.data:Newick"/>
     <sniffer type="galaxy.datatypes.data:Nexus"/>
+    <sniffer type="galaxy.datatypes.data:MaximumLikelihoodDistanceMatrix"/>
```

Comment:
Why have you added this. You don't have a sniffer for your filetype mldist, right?

Reply:
sorry, fixed